flows/engine/assets.go

```diff
@@ -190,6 +191,10 @@ func (s *assetServer) fetchAsset(url string, itemType assetType, isSet bool, use
 
 	defer response.Body.Close()
 
+	if log.GetLevel() >= log.DebugLevel {
```

Comment:
Why the if? It shouldn't log a debug statement at any level except debug and higher.